test/Spaces/terrain_warp.jl

```diff
@@ -61,27 +64,47 @@ function generate_base_spaces(
 
     # Generate Horizontal Space
     quad = Quadratures.GLL{npoly + 1}()
-    if ndims == 2
-        horzdomain = Domains.IntervalDomain(
-            Geometry.XPoint{FT}(xlim[1]),
-            Geometry.XPoint{FT}(xlim[2]);
-            periodic = true,
-        )
-        horzmesh = Meshes.IntervalMesh(horzdomain; nelems = helem)
-        horztopology = Topologies.IntervalTopology(comms_context, horzmesh)
-        hspace = Spaces.SpectralElementSpace1D(horztopology, quad)
-    elseif ndims == 3
-        horzdomain = Domains.RectangleDomain(
-            Geometry.XPoint{FT}(xlim[1]) .. Geometry.XPoint{FT}(xlim[2]),
-            Geometry.YPoint{FT}(xlim[1]) .. Geometry.YPoint{FT}(xlim[2]),
-            x1periodic = true,
-            x2periodic = true,
-        )
-        # Assume same number of elems (helem) in (x,y) directions
-        horzmesh = Meshes.RectilinearMesh(horzdomain, helem, helem)
-        horztopology = Topologies.Topology2D(comms_context, horzmesh)
-        hspace = Spaces.SpectralElementSpace2D(horztopology, quad)
-    end
+    horzdomain = Domains.IntervalDomain(
+        Geometry.XPoint{FT}(xlim[1]),
+        Geometry.XPoint{FT}(xlim[2]);
+        periodic = true,
+    )
+    horzmesh = Meshes.IntervalMesh(horzdomain; nelems = helem)
+    horztopology = Topologies.IntervalTopology(comms_context, horzmesh)
+    hspace = Spaces.SpectralElementSpace1D(horztopology, quad)
```

Comment:
Is `SpectralElementSpace1D` now GPU compatible?

Reply:
I don’t think so